src/control/server/mgmt_svc.go

```diff
@@ -42,7 +43,11 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-const instanceUpdateDelay = 500 * time.Millisecond
+const (
+	instanceUpdateDelay = 500 * time.Millisecond
+	minScmSize          = "16MB" // per storage target
+	minNvmeSize         = "1GB"  // per storage target
```

Comment:
Ditto what Kris said re: making the constants numeric. You can use consts from the humanize package to make this easy:
```go
const (
...
        minScmSize  = 16 * humanize.MByte
        minNvmeSize = 1 * humanize.GByte
)
```

Could maybe also make our own list of IEC/SI consts and put them in the storage package to insulate ourselves a bit from future changes, but I don't feel strongly either way.